### ISSUE TITLE
Recognize magic numbers in the files without external help

### ISSUE THREAD

**Issue by NobbZ** [NobbZ/file_info#4]
`file_info` should be able to recognize MIME-types on its own, without having to rely on the existence of `file` or it's implementation.

**Comment by d4rk5eed** [comment]
we could use http://www.garykessler.net/library/file_sigs.html or similiar signature table, and keep in-memory, for example ETS as binary keys, and search for signature on query. what d'u think?

**Comment by NobbZ** [comment]
Yeah, I already thought about creating a function during compile time that directly mathes on magic numbers. But hadn't found a good and reliable source so far which is machine readable.

Currently I do know only about the sources of `file`-tool itself, which I wasn't able to scan for the info necessary so far due to having no spare time anymore (job, studying and a newborn).

Now we have your given list on top. Perhaps you are able to create something? I'm open for a PR.
